pkg/operator/ceph/file/filesystem.go

```diff
@@ -249,3 +271,18 @@ func downFilesystem(context *clusterd.Context, clusterName, filesystemName strin
 	logger.Infof("Downed filesystem %s", filesystemName)
 	return nil
 }
+
+// GenerateMetadataPoolName is a helper function that generates a new name for the given metadata pool and returns the renamed pool.
+func GenerateMetadataPoolName(fsName string, metadataPool *model.Pool) *model.Pool {
+	metadataPool.Name = fmt.Sprintf("%s-%s", fsName, metaDataPoolSuffix)
+	return metadataPool
+
+}
+
+// GenerateDataPoolNames is a helper method that generates a new name for each pool in the input list suffixed with an appropriate index number, and it returns the renamed pool list
+func GenerateDataPoolNames(fsName string, dataPools []*model.Pool) []*model.Pool {
```

Comment:
Please return `[]string` from this method rather than updating the `dataPools` that are passed in.

Reply:
Done